x-pack/plugins/ml/public/application/explorer/anomaly_timeline.tsx

```diff
@@ -305,68 +285,84 @@ export const AnomalyTimeline: FC<AnomalyTimelineProps> = React.memo(
           >
-            {showOverallSwimlane && (
-              <SwimlaneContainer
-                filterActive={filterActive}
-                maskAll={maskAll}
-                timeBuckets={timeBuckets}
-                swimlaneCellClick={swimlaneCellClick}
-                swimlaneData={overallSwimlaneData as OverallSwimlaneData}
-                swimlaneType={'overall'}
-                selection={selectedCells}
-                swimlaneRenderDoneListener={swimlaneRenderDoneListener}
-                onResize={(width) => explorerService.setSwimlaneContainerWidth(width)}
-              />
-            )}
+            <SwimlaneContainer
+              filterActive={filterActive}
+              maskAll={maskAll}
+              timeBuckets={timeBuckets}
+              swimlaneCellClick={swimlaneCellClick}
+              swimlaneData={overallSwimlaneData as OverallSwimlaneData}
+              swimlaneType={'overall'}
+              selection={selectedCells}
+              swimlaneRenderDoneListener={swimlaneRenderDoneListener}
+              onResize={(width) => explorerService.setSwimlaneContainerWidth(width)}
+              isLoading={loading}
+              noDataWarning={<NoOverallData />}
+            />
           </div>
 
+          <EuiSpacer size="m" />
+
           {viewBySwimlaneOptions.length > 0 && (
             <>
-              {showViewBySwimlane && (
-                <>
-                  <EuiSpacer size="m" />
-                  <div
-                    className="ml-explorer-swimlane euiText"
-                    onMouseEnter={onSwimlaneEnterHandler}
-                    onMouseLeave={onSwimlaneLeaveHandler}
-                    data-test-subj="mlAnomalyExplorerSwimlaneViewBy"
-                  >
-                    <SwimlaneContainer
-                      filterActive={filterActive}
-                      maskAll={
-                        maskAll &&
-                        !hasMatchingPoints({
-                          filteredFields,
-                          swimlaneData: viewBySwimlaneData,
-                        })
+              <>
+                <div
+                  className="mlExplorerSwimlane euiText"
+                  onMouseEnter={onSwimlaneEnterHandler}
+                  onMouseLeave={onSwimlaneLeaveHandler}
+                  data-test-subj="mlAnomalyExplorerSwimlaneViewBy"
+                >
+                  <SwimlaneContainer
+                    filterActive={filterActive}
+                    maskAll={
+                      maskAll &&
+                      !hasMatchingPoints({
+                        filteredFields,
+                        swimlaneData: viewBySwimlaneData,
+                      })
+                    }
+                    timeBuckets={timeBuckets}
+                    swimlaneCellClick={swimlaneCellClick}
+                    swimlaneData={viewBySwimlaneData as ViewBySwimLaneData}
+                    swimlaneType={'viewBy'}
```

Comment:
Was the same in previous code, but could use `SWIMLANE_TYPE.VIEW_BY`.

Reply:
changed in 1370083a8647f01557d0dcee6f9d169e51283b3c